_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
+It should be emphasized that this is characteristic for a peer shared RelationshipAttribute.
+In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined.
+This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer.
 
-As already mentioned, the creation of a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) corresponds to the creation of one LocalAttribute for its `owner` and one [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for the peer with whom the `owner` has established a Relationship. The peer shared RelationshipAttribute is the peer's LocalAttribute and therefore forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set. Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified. In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined. This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+#### Own shared and peer shared ThirdPartyRelationshipAttributes
 
-Sometimes a peer shared RelationshipAttribute is referred to as a peer shared ThirdPartyRelationshipAttribute. This is the case if it can be interpreted as a counterpart of an own shared ThirdPartyRelationshipAttribute. Note that the `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to its owner.
-{: .notice--info}
+Note that it is possible to share the underlying RelationshipAttribute of an [own shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) with peers who are not involved in the Relationship in which the RelationshipAttribute exists, provided that the `confidentiality` of the RelationshipAttribute is not `"private"`.
+The sharing of a RelationshipAttribute by its `owner` with such a peer in turn leads to the creation of another own shared RelationshipAttribute in the wallet of the `owner`, which contains the Address of this peer within its `shareInfo.peer` property.
```

Comment:
"The sharing of a RelationshipAttribute by its owner with such a **third party**"

Reply:
I am in favor of avoiding the use of the word "third party" because, depending on the perspective, it differs who the third party is. To avoid confusion, I would therefore only refer to "peers" instead of "third parties" for the time being.